adopting-bitcoin/2022/how-to-get-started-contributing-sustainably-to-bitcoin-core.md

```diff
@@ -1,620 +1,700 @@
-It's helpful to not be the first person to get angry.
-It's helpful to read, say, meditations by Marcus Aurelius.
-And I also prefer people who say their position once without flooding over and over again and coming back and attacking anyone who disagrees with them.
-There are characters like that in space.
-That's not my cup of tea, and that's how I prefer to deal with it.
-Like, for example, the mempool full RBF recent discussions.
-There was one person who was just pretty much flooding all the threads everywhere on all the social networks and on the PR.
-My personal opinion is that's, well, maybe they got what they wanted.
-But I don't think that's the right way to have the debate.
-I think people should think carefully, respond when needed.
-But It's not always necessary to respond, especially to some sort of low kind of attack or criticism.
-If what you've thought and said is your position, that's enough.
-That's my position.
-
-Speaker 1: 00:23:28
-
-So perhaps, I mean, abstracting away from particular arguments and debates and things.
-What is the right way to reason and debate about these things?
-As I understand, there's a preference then for speaking about objective data or reasoned arguments.
-Is that the right way that people should debate things or what is the right way?
-
-Speaker 2: 00:23:51
-
-Well in theory this is about rough content consensus right a rough consensus isn't voting There's an article on it written back in the 90s by the internet standards committee back in the day, where rough consensus is humming or something like that, where basically everyone needs to feel heard.
-And I thought it was important on a process like the taproot activation one, which drew out for months and months.
-But some people got very upset about that and I wasn't upset at all because in my opinion it was necessary for all the people who had something to say to be able to say it and to feel heard.
-It's important for people to feel that validation, okay I was able to get out what I wanted to say.
-And I think that's fine.
-I mean, if people want to say something, they can.
-At the same time, there's a delicate balance.
-For example, in the recent Mempool full RBF, there's a whole bunch of people who have, say, an 80% understanding of the topic.
-Some people have a 90%, some people maybe a 95%.
-And there's maybe two or three people who I would say have a 99.9% understanding of the topic.
-The deepest, most experienced, long-term, active researchers and contributors in the space, in my opinion, is that they were shouted down, perhaps.
-People weren't listening to the most thoughtful and nuanced discussion, necessarily.
-Maybe they were.
-That's just my impression.
-My opinion is we shouldn't break things if we don't need to.
-We should be careful.
-But I'm not one of the young fiery contributors.
-I believe we should be prudent and reasonable.
+## Introduction
+
+    Host: 00:00:00
```

Comment:
We don't need those extra spaces at the beginning of each sentence.
```suggestion
Host: 00:00:00
```

Reply:
I deleted all the unnecessary whitespace.